kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UserTaskServicesClientImpl.java

```diff
@@ -870,6 +870,37 @@ public TaskInstance findTaskById(Long taskId) {
     }
 
     @Override
+    public List<TaskSummary> findTasksAssignedAsPotentialOwner(String userId, String filter, List<String> status, Integer page, Integer pageSize) {
```

Comment:
it can't. Be reused as it includes additional parameter, so the choice is between duplicate a bit or including additional if statements to check if filter is given or not. This is exspecially important for jms part that relies on number of arguments to select proper method server side